controller_manager/src/controller_manager_parameters.yaml

```diff
@@ -7,6 +7,15 @@ controller_manager:
   }
 
   hardware_components_initial_state:
+    not_loaded: {
```

Comment:
```suggestion
    unloaded: {
```
Can we name this `unloaded` instead of `not_loaded`, in the different parts of code

Reply:
I am open to it. Althrough for me personaly, `not_loaded` is clearer, `unloaded` is more for `unload` (verb) - but can change, no problem.

@bmagyar, @christophfroehlich what do you think about this?

Reply:
Sure. I don't have a strong opinion here. If they are fine, all good :)